annotator-core/build.gradle

```diff
@@ -50,7 +50,7 @@ dependencies {
 }
 
 // Should be the latest supporting version of NullAway.
-def NULLAWAY_TEST = "0.10.5"
+def NULLAWAY_TEST = "0.10.6-SNAPSHOT"
```

Comment:
I guess we need to wait on a new NullAway release before landing this one?

Reply:
Yes, we do. Please let me know if you would like me to update CI or gradle scripts to use locally snapshots for CI to pass for now. Thank you.

Reply:
I don't think we should use snapshots in CI.  Is the idea that we will only land this change after cutting a new NullAway release and updating?

Reply:
@msridhar Yes, we can land it once the new NullAway releases. I will keep checking status of changes locally.